ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/newsletter/clientlibs/less/newsletter.less

```diff
@@ -4,7 +4,7 @@
     h3{
         font-weight: normal;
         margin-bottom: 10px;
-         margin-top: 100px;
+          margin-top: 100px;
```

Comment:
what is the change here

Reply:
it was previously added but later on some changes required so i  deleted it but asked to retain these changes so again added